pybikes/encicla.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2015, Eduardo Mucelli Rezende Oliveira <edumucelli@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+import codecs
+
+from lxml import etree
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils, exceptions
+
+__all__ = ['Encicla', 'EnciclaStation']
+
+parse_methods = {
+  'json': 'get_json_stations',
+}
+
+class Encicla(BikeShareSystem):
+
+  sync = True
+
+  meta = {
+    'system': 'Encicla',
+    'company': 'Sistema de Bicicletas Públicas del Valle de Aburrá'
+  }
+
+  def __init__(self, tag, feed_url, meta, format):
+    super(Encicla, self).__init__(tag, meta)
+    self.feed_url = feed_url
+    self.method = format
+
+  def update(self, scraper = None):
+    if scraper is None:
+      scraper = utils.PyBikesScraper()
+
+    if self.method not in parse_methods:
+      raise Exception(
+        'Extractor for method %s is not implemented' % self.method )
+
+    self.stations = eval(parse_methods[self.method])(self, scraper)
+
+def get_json_stations(self, scraper):
+  data = json.loads(scraper.request(self.feed_url))
+  stations = []
+  
+  for station in data['stations']:
+    for item in station['items']:
+      station = EnciclaStation.from_json(item)
+      stations.append(station)
```

Comment:
Being a json feed trivial to consume there's no need to create an special EnciclaStation object.